odc/stac/_eo3converter.py

```diff
@@ -13,7 +13,12 @@
 import pystac.errors
 import pystac.item
 from datacube.index.eo3 import prep_eo3
-from datacube.index.index import default_metadata_type_docs
+
+try:
+    from datacube.index.index import default_metadata_type_docs  # type: ignore
```

Comment:
What are the conditions under which `index.default_metadata_type_docs` does not import?

Reply:
when running against `develop` branch of datacube.